crates/exex/types/src/head.rs

```diff
@@ -0,0 +1,9 @@
+use reth_primitives::BlockNumHash;
+
+#[allow(clippy::doc_markdown)]
```

Comment:
do we need this?

Reply:
ye, because of `ExEx` :/ `clippy::doc_markdown` forbids the camelcase outside the backticks